geoh5py/objects/object_base.py

```diff
@@ -344,7 +346,8 @@ def find_or_create_type(cls, workspace: Workspace, **kwargs) -> ObjectType:
 
         :return: The ObjectType instance for the given object class.
         """
-        return ObjectType.find_or_create(workspace, cls, **kwargs)
+        kwargs["entity_class"] = cls
```

Comment:
Same comment as above for the Groups, we should leave it as an explicit input since it is used directly in the `find_or_create` method